src/main/java/training/subsystems/roller/Roller.java

```diff
@@ -25,7 +25,7 @@ public static void init() {
 	}
 
 	public boolean isAtVelocity(double targetVelocity) {
-		return Math.abs(targetVelocity - motor.getEncoder().getVelocity()) == 0.0;
+		return Math.abs(targetVelocity - motor.getEncoder().getVelocity()) <= RollerConstants.TOLERANCE.getRadians();
```

Comment:
targetVelocity should be Rotation2d
you also should write a getVelocity function that returns Rot2d
and bcus tolerance is rot2d also you have three rot2ds so no need for .getsomething and you can do it whatever u want in ur code